iterx.go

```diff
@@ -257,27 +258,39 @@ func (iter *Iterx) structScan(value reflect.Value) bool {
 		panic("value must be a pointer")
 	}
 
+	casResponse := false
```

Comment:
This only work for the 1st row!

It would be better to prepend `iter.fields` with nil to accomodate for applied and and init iter.values[0] in the if branch below.

Reply:
I'd rename that to just cas btw.